### ISSUE TITLE
Windows 11 Start Menu Styler text cursor is displaced from its original location inside the text box

### ISSUE THREAD

**Issue by aashish108** [ramensoftware/windhawk-mods#733]
Hey there. There's an issue with the Windows 11 Start Menu Styler, the text cursor is off when the Windows menu is initially opened.

**Mod version** 1.1.3
Nothing of note in the logs
**Theme** SideBySide23
**Control styles**:
Windows.UI.Xaml.Controls.Grid#ShowMoreSuggestions Visibility=Collapsed
Windows.UI.Xaml.Controls.Grid#SuggestionsParentContainer Visibility=Collapsed
Windows.UI.Xaml.Controls.Grid#TopLevelSuggestionsListHeader Visibility=Collapsed

**Mod Settings**
```
{"theme":"SideBySide2","controlStyles[0].target":"Windows.UI.Xaml.Controls.Grid#ShowMoreSuggestions","controlStyles[0].styles[0]":"Visibility=Collapsed","resourceVariables[0].variableKey":"","resourceVariables[0].value":"","controlStyles[1].target":"Windows.UI.Xaml.Controls.Grid#SuggestionsParentContainer","controlStyles[1].styles[0]":"Visibility=Collapsed","controlStyles[2].target":"Windows.UI.Xaml.Controls.Grid#TopLevelSuggestionsListHeader","controlStyles[2].styles[0]":"Visibility=Collapsed","controlStyles[3].target":"StartMenu.PinnedList","controlStyles[3].styles[0]":"Height=504"}
```

cc: @m417z 

![image](https://github.com/ramensoftware/windhawk-mods/assets/7046226/cce0547e-efad-4889-b640-9101bdebadfe)

![image](https://github.com/ramensoftware/windhawk-mods/assets/7046226/6f572a1f-fab1-4392-aa41-e3cd29b26331)

Edit: SideBySide theme seems to not have this issue though! Its just SideBySide2.

**Comment by LoneWolf993** [comment]
How did you remove the recommended section?

**Comment by aashish108** [comment]
> How did you remove the recommended section?

It's in the readme. You need 3 custom control styles:

target: `Windows.UI.Xaml.Controls.Grid#ShowMoreSuggestions`
style: `Visibility=Collapsed`

target: `Windows.UI.Xaml.Controls.Grid#SuggestionsParentContainer`
style: `Visibility=Collapsed`

target: `Windows.UI.Xaml.Controls.Grid#TopLevelSuggestionsListHeader`
style: `Visibility=Collapsed`

target: `StartMenu.PinnedList`
style:`Visibility=Height=504`

**Comment by m417z** [comment]
Fixed in Windows 11 Start Menu Styler v1.1.5.
